client/app/scripts/superdesk-dashboard/views/workspace-sidenav.html

```diff
@@ -1,26 +1,33 @@
 <ul>
     <li ng-class="{active: currentRoute.href == '/workspace'}">
-        <a href="#/workspace" class="big-icon-dashboard" translate></a>
+        <a href="#/workspace" class="big-icon-dashboard"
+            title="{{ :: 'Dashboard' | translate }}"></a>

```

Comment:
I'm not using tooltips here because need to check it's performance implications. it's doing some dom manipulation every digest cycle